TypeScript/confirmation.ts

```diff
@@ -0,0 +1,139 @@
+//@ts-nocheck
+/** 
+ * Asks the user for a confirmation message before executing the command
+ * 
+ * @author @HighArcs [<@504698587221852172>]
+ * @version 1.0.0
+ * @example
+ * ```ts
+ * import { confirmation } from "../plugins/confirmation";
+ * import { CommandType, commandModule } from "@sern/handler";
+ *
+ * // YOU MUST HAVE GUILD MESSAGE REACTION INTENTS ON YOUR CLIENT!!!!
+ * export default commandModule({
+ *  type : CommandType.Both
+ *  plugins: [confirmation()],
+ * 	execute: (ctx, args) => {
+ * 		ctx.reply('hola');
+ * 	}
+ * })
+ * ```
+ **/
```

Comment:
I swear you're gonna be so angry on me right now
```suggestion
 */
```

Reply:
😟I'll fix one sec